.github/workflows/checkbox-promote-beta-to-candidate/job.template

```diff
@@ -35,24 +35,23 @@ test_data:
     # install checkbox runtime on device (snap or deb)
     if [[ "$SNAP_FLAG" == "true" ]]; then
         echo "Installing checkbox runtime on device (from snap)"
-        CHANGE_ID=$(_run sudo snap install $CHECKBOX_RUNTIME --no-wait --channel=latest/$CHANNEL)
-        wait_for_snap_change_id $CHANGE_ID
+        _run_retry sudo snap install $CHECKBOX_RUNTIME --channel=latest/$CHANNEL
```

Comment:
I should have removed `run_retry`, that's exactly what `retry` is for. Would you mind trying something like this?
```
retry --times 5 --  _run sudo snap install $CHECKBOX_RUNTIME --channel=latest/$CHANNEL
```
Unless you find `_run_retry` very convenient.

Reply:
Updated